docs/testslide_dsl/async_support/index.rst

```diff
@@ -184,3 +184,36 @@ Python's default threshold for triggering this event loop lock up failure is **1
 
   Finished 1 example(s) in 1.0s
     Successful: 1
+
+Leaked Tasks
+^^^^^^^^^^^^
+
+If your async code creates a task in the asyncio loop, but finished before that task has ended (ex. you forgot to await for it), testslide will catch it and fail the test.
+
+This is enabled by default for async tests, but to get that behaviour also when running async code from sync tests, for example:
```

Comment:
Shouldn't `self.async_run`  encapsulate all event loop health checks we have, and not only the leaked tasks?